editor/src/components/canvas/canvas-utils.spec.browser.ts

```diff
@@ -617,82 +553,16 @@ describe('updateFramesOfScenesAndComponents - pinMoveChange -', () => {
 
     expect(getPrintedUiJsCode(renderResult.getEditorState())).toEqual(
       makeTestProjectCodeWithSnippet(
-        `<View style={{ ...props.style }} layout={{ layoutSystem: 'pinSystem' }} data-uid='aaa'>
+        `<View style={{ ...props.style }} data-uid='aaa'>
         <View
           style={{
             backgroundColor: '#0091FFAA',
+            position: 'absolute',
             left: '15%',
             top: '31.3%',
             right: '5%',
             bottom: '8.8%',
           }}
-          layout={{ layoutSystem: 'pinSystem' }}
-          data-uid='bbb'
-        />
-      </View>`,
-      ),
-    )
-  })
-
-  it('TLCxCy pin change works', async () => {
-    const renderResult = await renderTestEditorWithCode(
-      makeTestProjectCodeWithSnippet(
-        `<View style={{ ...props.style }} layout={{ layoutSystem: 'pinSystem' }} data-uid='aaa'>
-        <View
-          style={{ backgroundColor: '#0091FFAA', left: 52, top: 61 }}
-          layout={{ layoutSystem: 'pinSystem', centerX: 100, centerY: 100 }}
-          data-uid='bbb'
-        />
-      </View>`,
-      ),
-    )
-
-    const pinChange = pinMoveChange(TP.instancePath(TestScenePath, ['aaa', 'bbb']), {
-      x: -32,
-      y: -41,
-    } as CanvasVector)
-
-    await renderResult.dispatch([setCanvasFrames([pinChange], false)], true)
-
-    expect(getPrintedUiJsCode(renderResult.getEditorState())).toEqual(
-      makeTestProjectCodeWithSnippet(
-        `<View style={{ ...props.style }} layout={{ layoutSystem: 'pinSystem' }} data-uid='aaa'>
-        <View
-          style={{ backgroundColor: '#0091FFAA', left: 20, top: 20 }}
-          layout={{ layoutSystem: 'pinSystem', centerX: 68, centerY: 59 }}
```

Comment:
WOW, I thought that this was killed a _long_ time ago! Apparently this is still kicking on master? @enidemi while you are spring cleaning, could you murder the code responsible for updating centerX and centerY values? :D

Reply:
Because Views supported layout props, centerX and centerY worked when you just typed it to the code editor. There are some TODOs left in the code to remove layoutsystem prop, it should be the next step.

Reply:
yeah but apparently resizing properly updates centerX and centerY! that's a good chunk of code making the resize logic very complicated, I think it'd be worth to make a follow-up PR and delete half of that

Reply:
👍